backend/sequelizeManager.js

```diff
@@ -90,31 +90,56 @@ export class SequelizeManager {
     }
 
     createConnection(configuration) {
-
-        const {
+        let {
             username, password, database, port,
             dialect, storage, host
             } = configuration;
 
-        this.log(`Creating a connection for user ${username}`, 1);
-
-        this.connection = new Sequelize(database, username, password, {
+        let options = {
             dialect,
             host,
             port,
             storage
-        });
+        };
+
+        this.log(`Creating a connection for user ${username}`, 1);
+
+        let subDialect = null;
+        if (this.connection) {
+            subDialect = this.connection.config.subDialect || null;
+        }
+        let redshiftOptions = null;
+
+        if ((dialect === 'redshift') || (subDialect === 'redshift')) {
+            subDialect = 'redshift';
```

Comment:
This is true when it's either the first Redshift connection the user is doing or when the user wants to change the database to which he/she is connected. i.e. is connected to database1 with Redshift and wants to connect to database2 with Redshift for same credentials.